app/models/agents/jira_agent.rb

```diff
@@ -84,7 +85,7 @@ def check
 
   private
     def request_url(jql, start_at)
-      "#{interpolated[:jira_url]}/rest/api/2/search?jql=#{CGI::escape(jql)}&fields=*all&startAt=#{start_at}"
+      "#{interpolated[:jira_url]}/rest/api/2/search?jql=#{CGI::escape(jql)}&expand=changelog&fields=*all&startAt=#{start_at}"
```

Comment:
Does the changelog change the shape of the parts of the Event that were already present before this change?

The tests probably need to be adjusted.